frontend/public/locales/en/bma.json

```diff
@@ -29,6 +29,7 @@
   "editBareMetalAsset.button.submit":"Apply",
 
   "bareMetalAsset.emptyState.title":"No bare metal assets found",
+  "bareMetalAsset.emptyState.subtitle": "<p>You don't have any bare metal assets.</p>Click the <bold>Create bare metal asset</bold> or <bold>Import bare metal assets</bold> button to create your resource",
```

Comment:
I think these don't need the paragraph separation `<p />`

Reply:
At first I thought I needed <p> to use <br>, then I noticed <p> will create a newline all by itself after closing...best to use <br> though.